E2_ESRGAN/lib/phase_1.py

```diff
@@ -0,0 +1,68 @@
+from functools import partial
+import tensorflow as tf
+from lib import settings, utils, model, dataset
+
+class Model:
+  def __init__(self, **kwargs):
+  """ PSNR model training on L1 Loss to warmup the Generator
```

Comment:
Define the acronym first time you use it :-)

Reply:
Done :)